### PR TITLE
feat: redirect using cf ip geolocation

### PR DESCRIPTION
**Pull request by elbotho**
merging into staging now equals deploying to staging, right?

### CODE REVIEW

**Comment by kulla** [comment]
> merging into staging now equals deploying to staging, right?

Right

File: src/redirects.ts [hunk 1]
```diff
@@ -224,4 +224,15 @@ export async function redirects(request: Request, env: CFEnvironment) {
...
+  if (
+    isInstance(url.subdomain) &&
+    url.subdomain === Instance.De &&
+    url.pathnameWithoutTrailingSlash === '/pruefungen'
+  ) {
+    const regionSlug =
+      request.cf?.regionCode === 'NI' ? 'niedersachsen' : 'bayern'
+    url.pathname = `/pruefungen/${regionSlug}`
+    return url.toRedirect(302)
+  }
```
Comment (kulla) [inline src/redirects.ts:237]: Unit tests would be great

**Review by kulla — APPROVED** [review]
I'll deploy it, so that it can be tested

**Comment by elbotho** [comment]
Thank you @kulla!

**Pull request merged**